card/lib/cardio.rb

```diff
@@ -76,8 +77,22 @@ def set_paths paths
       add_path "db/seeds", :with => "db/seeds.rb"
 
       add_path 'config/initializers',  :glob => '**/*.rb'
-      paths['config/initializers'] << "#{gem_root}/mod/**{,/*/**}/initializers"
-      paths['config/initializers'] << "#{root}/mod/**{,/*/**}/initializers"
+
+    end
+
```

Comment:
I'm still confused about all the config and paths objects, and when they are loaded.
How did you get this to work then?  Did you copy stuff or some other shim?